app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -156,6 +156,9 @@ export default {
     // Status columns in the table
     getCurrentStatus(record) {
       if (record.draft) {
+        if (record.submissionStatus[0].code === 'REVISING') {
+          return 'REVISING';
+        }
         return 'DRAFT';
       } else {
         // Current status is most recent status (top in array, query returns in status created desc)
```

Comment:
If we need the status now in both parts of the conditional, we'd still want to gaurd against exceptions, so I'd refactor this to 
```
      // Current status is most recent status (top in array, query returns in status created desc)
      const status = record.submissionStatus && record.submissionStatus[0]
          ? record.submissionStatus[0].code
          : 'N/A';
      if (record.draft && status !== 'REVISING') {
        return 'DRAFT';
      } else {
        return status
      }
 ```